### PR TITLE
Refactor dnd component

### PR DESCRIPTION
**Pull request by danielcsato**

### CODE REVIEW

File: frontend/src/components/Columns.jsx [hunk 1]
```diff
@@ -19,22 +19,22 @@ function Columns() {
       setDragging(true);
     }, 0);
   };
+
   const handleDragEnter = (e, targetItem) => {
-    console.log('Drop target: ', targetItem);
     if (dragItemNode.current !== e.target) {
-      console.log('old: ', list);
+      const targetGroupIndex = targetItem.groupIndex;
```
Comment (Mandlee) [inline frontend/src/components/Columns.jsx:25]: https://wesbos.com/destructuring-renaming

```suggestion
      const { groupIndex : targetGroupIndex } = targetItem;
```

File: frontend/src/components/Columns.jsx [hunk 2]
```diff
@@ -43,42 +43,33 @@ function Columns() {
...
-    if (
-      dragItem.current.grpI === item.grpI &&
-      dragItem.current.itemI === item.itemI
-    ) {
-      return 'dnd-item current';
-    }
-    return 'dnd-item';
-  };
+  const getClasses = (item) =>
+    dragging && dragItem.current.groupIndex === item.groupIndex && dragItem.current.itemIndex === item.itemIndex
```
Comment (Mandlee) [inline frontend/src/components/Columns.jsx:47]: I'd use object destructure

**Review by Mandlee — CHANGES_REQUESTED** [review]

File: frontend/src/components/Columns.jsx (hunk 1 above)
Comment (danielcsato) [reply frontend/src/components/Columns.jsx:20]: https://github.com/hegedus-adam/kanban-board/pull/15/commits/af2c91558cf52c265e7a384e53b9bf158efc936b

File: frontend/src/components/Columns.jsx (hunk 2 above)
Comment (danielcsato) [reply frontend/src/components/Columns.jsx:42]: https://github.com/hegedus-adam/kanban-board/pull/15/commits/af2c91558cf52c265e7a384e53b9bf158efc936b

**Pull request merged**